packages/@aws-cdk/aws-appmesh/lib/virtual-gateway-listener.ts

```diff
@@ -36,7 +36,7 @@ interface VirtualGatewayListenerCommonOptions {
    *
    * @default - none
    */
-  readonly tlsCertificate?: TlsCertificate;
+  readonly tls?: TlsListener;
```

Comment:
This is a breaking change so you'll have to note that in the PR description.

Reply:
Got it. Will ensure to add this when submitting the PR to main repo.